### PR TITLE
Gradual WIP changes

### PR DESCRIPTION
**Pull request by erelson**
Will push to my fork for now...

### CODE REVIEW

File: fetch_binary_drivers/package.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
   <!-- system dependencies which the drivers dyanmically link against -->
   <depend>boost</depend>
   <depend>curl</depend>
-  <depend>python</depend>
+  <depend>python3</depend>
```
Comment (mikeferguson) [inline fetch_binary_drivers/package.xml:42]: If you want the same branch to support both noetic (and earlier) you can upgrade to package format 3 and use conditionals (see https://github.com/fetchrobotics/robot_controllers/blob/melodic-devel/robot_controllers/package.xml#L29)

Comment (erelson) [reply fetch_binary_drivers/package.xml:42]: I had been planning on keeping melodic_devel for any melodic releases.  I'm wary of complexity (which in this case is anything new beyond the minimum needed 😬 ) given my limited time to test much.

File: fetch_system_config/root/lib/udev/rules.d/99-logitechF710.rules [hunk 2]
```diff
@@ -1,4 +0,0 @@
-KERNEL=="js?", SUBSYSTEM=="input", ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c219", SYMLINK+="logitechF710"
```
Comment (cjds) [inline fetch_system_config/root/lib/udev/rules.d/99-logitechF710.rules:1]: No need for these anymore?

Comment (erelson) [reply fetch_system_config/root/lib/udev/rules.d/99-logitechF710.rules:1]: We could walk back from this, but we've never shipped the Logitech F710 controller or similar with the research robots, and so, one less thing to support here, right?

**Review by cjds — APPROVED** [review]

File: fetch_bringup/launch/include/teleop.launch.xml [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
   <arg name="joy_device" default="/dev/fetch_joy"/>
   <arg name="ps4" default="false"/>
+  <arg name="is_fetch" default="false"/>
```
Comment (JeffWilson7) [inline fetch_bringup/launch/include/teleop.launch.xml]: Doesn't look like this is used?

**Review by JeffWilson7 — APPROVED** [review]
Had one question...

File: fetch_bringup/launch/include/teleop.launch.xml (hunk 3 above)
Comment (erelson) [reply fetch_bringup/launch/include/teleop.launch.xml]: Ahh, missed that in cleanup. Thanks!

At one point I had a bunch of added logic, before realizing why things weren't working at one point.  Why things weren't working: I ran freight.launch which sets /teleop/is_fetch to false, and then ran fetch.launch again later, and I could only teleop the base.... And that's why I now have the is_fetch = True line later in this launch file.

**Review by cjds — APPROVED** [review]
LGTM

**Comment by erelson** [comment]
Note to self for future (¯\_(ツ)_/¯): Squashing the commits wasn't helpful when I generated the changelogs.  (also changelog generation respects CATKIN IGNORE files; TIL)

**Pull request merged**